src/views/List.jsx

```diff
@@ -1,12 +1,25 @@
 import { ListItem } from '../components';
 import { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
 
 /** List component that displays items in a user's shopping cart  */
-export function List({ data }) {
+export function List({ data, loading }) {
 	const [filteredList, setFilteredList] = useState([]);
 	const [filterInput, setFilterInput] = useState('');
 
-	/* Use handler to change the state of filterInput  and convert all items to lowercase to facilitate a more thorough search */
+	/* Declare navigate for view redirection */
+	const navigate = useNavigate();
+
+	/* This function will return a boolean value of whether the list
+	is empty or not */
+	const checkForEmptyList = () => {
+		/* return true if list is empty and false if list contains items */
+		const shoppingListArr = data.map((item) => item.name);
+		return shoppingListArr.includes(undefined) ? true : false;
```

Comment:
@alex-andria / @Bustamor this will evaluate to `true` since the blank item will return undefined here

Reply:
Good catch! Resolved with `filter()` and then a length check later instead of includes.